src/styles/blocks/testimonial.scss

```diff
@@ -0,0 +1,50 @@
+.testimonial {
+
+  &__photo-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 24px;
+    margin-bottom: 28px;
+
+    @include desktop {
+      margin-bottom: 32px;
+    }
+  }
+
+  &__photo {
+    grid-column: 2 / 4;
+    width: 100%;
+    border-radius: 50%;
+    transition-duration: 0.5s;
+
+    @include desktop {
+      &:hover {
+        transform: scale(2);
+      }
+    }
```

Comment:
let's scale it to less extend, now the photo is too big and it covers the title
<img width="636" alt="Screenshot 2021-09-06 at 11 28 43" src="https://user-images.githubusercontent.com/57598825/132186661-94d848ad-8a6b-473c-8f83-8aa7b1211f98.png">

Reply:
changed from scale 2.0 to scale 1.2